src/helm/benchmark/run_specs.py

```diff
@@ -2550,7 +2569,9 @@ def construct_run_specs(spec: ObjectSpec) -> List[RunSpec]:
 
     def alter_run_spec(run_spec: RunSpec) -> RunSpec:
         try:
-            model = get_model(run_spec.adapter_spec.model)
+            deployment: ModelDeployment = get_model_deployment(run_spec.adapter_spec.model_deployment)
+            model_name: str = deployment.model_name or deployment.name
```

Comment:
Might be good to raise a `ValueError` here if `model_deployment.model_name` does not equal `adapter_spec.model`.

Reply:
Well not necessarily. `adapter_spec.model` could be set to `together/llama-7b` and now `model_deployment_model_name` would be set to `meta/llama-7b`.

Reply:
Actually this example was not good. A good example is `together/flan-t5-xxl` which is now called `google/flan-t5-xxl` and deployed by `together/flan-t5-xxl`.